src/raftstore/store/store.rs

```diff
@@ -202,7 +202,8 @@ pub struct Store<T: Transport, C: PdClient + 'static> {
     pending_regions: Vec<metapb::Region>,
     split_check_worker: Worker<SplitCheckTask>,
     region_worker: Worker<RegionTask>,
-    compact_worker: Worker<CompactTask>,
+    raft_compact_worker: Worker<CompactTask>,
+    manual_compact_worker: Worker<CompactTask>,
```

Comment:
`manual_compact_worker` is confusing. It's hard to figure out what does this worker want to compact and what does compact mean.